tests/test_dataset.py

```diff
@@ -310,80 +361,92 @@ def test_graph_standardize(self):
                 if vals.ndim == 1: # features with only one channel
                     arr = []
                     for entry_idx in range(len(dataset)):
-                        arr.append(dataset.get(entry_idx).x[:, tensor_idx])
+                        arr.append(dataset.get(entry_idx).x[:, tensor_idx]) 
                     arr = np.concatenate(arr)
                     features_dict[feat] = arr
                     tensor_idx += 1
                 else:
                     for ch in range(vals.shape[1]):
                         arr = []
                         for entry_idx in range(len(dataset)):
-                            arr.append(dataset.get(entry_idx).x[:, tensor_idx])
+                            arr.append(dataset.get(entry_idx).x[:, tensor_idx]) 
                         tensor_idx += 1
                         arr = np.concatenate(arr)
                         features_dict[feat + f'_{ch}'] = arr
 
-            for _, values in features_dict.items():
-
-                mean = values.mean()
-                dev = values.std()
-
-                assert -0.3 < mean < 0.3
-                # for one hot encoded features, with few data points it can happen that mean and std are not exactly 0 and 1
-                assert 0.7 < dev < 1.5
-
             # getting all edge features values
             tensor_idx = 0
-            features_dict = {}
             for feat in dataset.edge_features:
                 vals = grp[f"{Efeat.EDGE}/{feat}"][()]
                 if vals.ndim == 1: # features with only one channel
                     arr = []
                     for entry_idx in range(len(dataset)):
-                        arr.append(dataset.get(entry_idx).edge_attr[:, tensor_idx])
+                        arr.append(dataset.get(entry_idx).edge_attr[:, tensor_idx]) 
                     arr = np.concatenate(arr)
                     features_dict[feat] = arr
                     tensor_idx += 1
                 else:
                     for ch in range(vals.shape[1]):
                         arr = []
                         for entry_idx in range(len(dataset)):
-                            arr.append(dataset.get(entry_idx).edge_attr[:, tensor_idx])
+                            arr.append(dataset.get(entry_idx).edge_attr[:, tensor_idx]) 
                         tensor_idx += 1
                         arr = np.concatenate(arr)
                         features_dict[feat + f'_{ch}'] = arr
 
-            for _, values in features_dict.items():
-
-                mean = values.mean()
-                dev = values.std()
-
-                assert -0.2 < mean < 0.2
-                assert 0.8 < dev < 1.2
+            for key, values in features_dict.items():
+                if(key in features_transform):
+                    standardization = features_transform.get(key, {}).get('standardize')
+                    if standardization: #Feature contains in dictionary & standardization=True
```

Comment:
here you're not testing `hb_donors`, for which standardize is False. In order to test it, you can define another GraphDataset like `no_std_dataset` without defining a `features_transform` dict, and then you can verify:
1) that features of `dataset` which are in `features_transform` with standardize True have mean and dev as indicated (mean around 0 and dev around 1) and different mean and dev from the same features in `no_std_dataset`
2) that features of `dataset` which are in `features_transform` with standardize False or which are not in the dict have mean and dev equal to the ones in `no_std_dataset`